expr/function/function.go

```diff
@@ -231,7 +222,60 @@ func (i *Is) Call(args []zed.Value) (zed.Value, error) {
 		typ, err = i.zctx.LookupByValue(zvTypeVal.Bytes)
 	}
 	if err == nil && typ == zvSubject.Type {
-		return zed.True, nil
+		return zed.True
 	}
-	return zed.False, nil
+	return zed.False
+}
+
+// https://github.com/brimdata/zed/blob/main/docs/language/functions.md#quiet
+type Quiet struct{}
+
+func (q *Quiet) Call(_ zed.Allocator, args []zed.Value) *zed.Value {
+	val := args[0]
+	if val.IsMissing() {
+		return zed.Quiet
+	}
+	return &val
+}
+
+func newInt64(ctx zed.Allocator, native int64) *zed.Value {
+	return newInt(ctx, zed.TypeInt64, native)
+}
+
+func newInt(ctx zed.Allocator, typ zed.Type, native int64) *zed.Value {
+	//XXX we should have an interface to allocator where we can
+	// append into some new bytes; for now, the byte slice goes through GC.
+	return ctx.NewValue(typ, zed.EncodeInt(native))
+}
+
+func newUint64(ctx zed.Allocator, native uint64) *zed.Value {
+	return newUint(ctx, zed.TypeUint64, native)
+}
+
+func newUint(ctx zed.Allocator, typ zed.Type, native uint64) *zed.Value {
+	return ctx.NewValue(typ, zed.EncodeUint(native))
+}
+
+func newFloat64(ctx zed.Allocator, native float64) *zed.Value {
+	return ctx.NewValue(zed.TypeFloat64, zed.EncodeFloat64(native))
+}
+
+func newDuration(ctx zed.Allocator, native nano.Duration) *zed.Value {
+	return ctx.NewValue(zed.TypeDuration, zed.EncodeDuration(native))
+}
+
+func newTime(ctx zed.Allocator, native nano.Ts) *zed.Value {
+	return ctx.NewValue(zed.TypeTime, zed.EncodeTime(native))
+}
+
+func newString(ctx zed.Allocator, native string) *zed.Value {
+	return ctx.NewValue(zed.TypeString, zed.EncodeString(native))
+}
+
+func newError(ctx zed.Allocator, err error) *zed.Value {
+	return ctx.CopyValue(zed.NewError(err))
+}
+
+func newErrorf(ctx zed.Allocator, format string, args ...interface{}) *zed.Value {
+	return ctx.CopyValue(zed.NewErrorf(format, args...))
```

Comment:
Copying these is unnecessary since they their bytes.

Reply:
But it returns a zed.Value not *zed.Value